components/home/TabList.css

```diff
@@ -0,0 +1,8 @@
+.nav-style {
+  background-color: grey;
+  border: 2px solid #5cb85c;
+}
+
+.nav-pills, .nav-link {
+  border-radius: 0 !important;
```

Comment:
If we could have one `!important` that we did need for some reason (not sure how we could set that scenario up) and one that wasn't need that could be a good intermediate-senior indicator for CSS knowledge